admin/server.go

```diff
@@ -10,6 +10,12 @@ import (
 	tunnel "github.com/nknorg/nkn-tunnel"
 )
 
+var networkMemberIAccept []string
```

Comment:
I don't recommend using global variable

Reply:
I don't get a better and simple solution yet :(. It is a module unexported variable.